docs/source/teams/installation.rst

```diff
@@ -221,7 +221,10 @@ You can do this in any of the following ways:
 
 1. Configure/provide AWS credentials in accordance with the
 `boto3 <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials>`_
-python library.
+python library, such as via environment variable, AWS config file, or IAM role.
+
+For example, by setting the environment variables: ``AWS_ACCESS_KEY_ID`` and
```

Comment:
For context, we used to explicitly mention options like `AWS_ACCESS_KEY_ID` here, but at some point we removed them and instead just linked to `boto3`, with the *goal* of conveying to the reader that they can configure credentials in their environment in *any way* that boto3 supports, which includes access key, config file, web identity token, IAM role, and probably other concoctions they've invented since then.

Re-reading this documentation now, I can see why something caused you to revisit this; the way it currently reads it doesn't feel like I'm being drawn to click on the `boto3` link hard enough... And also we *do* explicitly list all the supported ways to work with Azure and MinIO below.

My concern is that we should either list every combination of environment variables, or none of them.

A failure mode is that customers make feature requests for us to support things like `AWS_WEB_IDENTITY_TOKEN_FILE`, which we do in fact already support.

For consistency with Azure/MinIO, perhaps we should just bring back the explicit list of credential types that are supported, together with a `note::` stating that there could be more?

Reply:
It was an internal ask by @lkaneda that env vars be documented. I added the credentials path variables for consistency but then realized she might also have meant she wanted `AWS_ACCESS_KEY_ID`. So put that with qualifier language like "such as ...", "for example ..." 

> For consistency with Azure/MinIO, perhaps we should just bring back the explicit list of credential types that are supported, together with a note:: stating that there could be more?

Sure sounds reasonable.

Reply:
Here's a list of ways we used to document here:

```
export AWS_ACCESS_KEY_ID=...
export AWS_SECRET_ACCESS_KEY=...
export AWS_SESSION_TOKEN=... (if applicable)
export AWS_DEFAULT_REGION=...

export AWS_ROLE_ARN=...
export AWS_WEB_IDENTITY_TOKEN_FILE=...
export AWS_ROLE_SESSION_NAME... (if applicable)
export AWS_DEFAULT_REGION=...

export AWS_SHARED_CREDENTIALS_FILE="/path/to/aws-credentials.ini"
export AWS_PROFILE=default  # optional

export AWS_CONFIG_FILE="/path/to/aws-config.ini"
export AWS_PROFILE=default  # optional
```

Reply:
Added the above with a note that it is just for convenience and subject to change.

![image](https://github.com/voxel51/fiftyone/assets/6363888/36d26abc-68d5-4edb-b942-425970584e3a)